tests/test_core.py

```diff
@@ -7333,8 +7333,6 @@ def post(filename):
   @no_wasm2js('TODO: source maps in wasm2js')
   @no_emterpreter
   def test_source_map(self):
-    if not jsrun.check_engine(NODE_JS):
-      self.skipTest('sourcemapper requires Node to run')
```

Comment:
NODE_JS is a basic requirement of emscripten, we don't support running without it, so no need to check for it.